cmake/features.cmake

```diff
@@ -68,3 +68,5 @@ Usage: -DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=/path/*.csv" OFF
                ALLOWED_VALUES ON OFF COLLECT)
 
 set(LINKCHECKER_PY "" CACHE FILEPATH "Path to linkchecker.py for documentation check")
+
+ie_option(HIGHLIGHT_ERRORS "Highlight errors and warnings during compile time" OFF)
```

Comment:
Maybe `ENABLE_ERROR_HIGHLIGHT` is better name.

Reply:
Fixed